### PR TITLE
week 1 hw

### PR DESCRIPTION
**Pull request by marcmwebdev**

### CODE REVIEW

**Comment by mikaelacurrier** [comment]
Hi Marc, please remember not to merge your pull requests until the reviewer has graded your assignment - they will do that for you

File: index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>portfolio-site</portfolio-site></title>
```
Comment (danyll39) [inline index.html:8]: work on clean code

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
+h1 {
+  margin-bottom: 60px;
+  style="text-decoration: underline;">
```
Comment (danyll39) [inline styles.css:3]: wrong CSS syntax

File: index.html (hunk 1 above)
Comment (marcmwebdev) [reply index.html:8]: Im confused, what you're referring to here.

File: styles.css (hunk 2 above)
Comment (marcmwebdev) [reply styles.css:3]: Thanks, not sure why I missed that... I'm pretty sure I copied it over from doing inline styles in html to start.

**Comment by danyll39** [comment]
Hi Marc,

No problem, you’ll get used to triple checking your code and looking out for little errors like that. It’s only the first week of homework! Good job so far!

All the best,
Ursala Lindsey

Let’s stay connected
LinkedIn
 <https://www.linkedin.com/in/ursala-lindsey>Github <https://github.com/danyll39>

**Comment by danyll39** [comment]
Marc,

There is a weird portfolio-site element tag in there, you could just write that as <title>portfolio-site</title> 



All the best,
Ursala Lindsey

Let’s stay connected
LinkedIn
 <https://www.linkedin.com/in/ursala-lindsey>Github <https://github.com/danyll39>

**Comment by marcmwebdev** [comment]
@danyll39 True , I have no idea how that happened. A title element doesn't need anything besides the title lol I must have been sleep coding or something.

**Comment by danyll39** [comment]
Haha that’s funny I do that from time to time!!

All the best,
Ursala Lindsey

Let’s stay connected
LinkedIn
 <https://www.linkedin.com/in/ursala-lindsey>Github <https://github.com/danyll39>

**Pull request merged**